src/ingestion-server/custom-resource/update-alb-rules/index.ts

```diff
@@ -55,6 +55,7 @@ type ResourceEvent = CloudFormationCustomResourceEvent;
 
 export const handler = async (event: ResourceEvent, context: Context) => {
   try {
+    logger.info('input event: ', { event });
```

Comment:
It's a redundancy log. We enabled `POWERTOOLS_LOGGER_LOG_EVENT: 'true',` to log the event of Lambda.

Reply:
Removed this line